Backend/src/middlewares/logger.ts

```diff
@@ -0,0 +1,59 @@
+import winston, { format } from "winston";
+import * as expressWinston from "express-winston";
+import path from "path"
+
+let requestLoggerTransports: Array<any> = [
+    //always save to file
+    new winston.transports.File({ filename: path.join(process.cwd(), "logs", "requests.log") })
+]
+let errorLoggerTransports: Array<any> = [
+    //always save to file
+    new winston.transports.File({ filename: path.join(process.cwd(), "logs", "error.log") })
+]
```

Comment:
So this is where it makes the file, where it stores the log, right? 
Where does this file go?

Reply:
root/log/filename